integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoTestUsingSparkStore.scala

```diff
@@ -325,4 +326,22 @@ class PrestoTestUsingSparkStore
 
   }
 
+  test("Test decimal unscaled converter") {
+    prestoServer
+      .execute(
+        "create table presto_spark_db.array_decimal(salary array(decimal(20,3)) ) with" +
+        "(format='CARBON') ")
+    copyStoreContents("array_decimal")
+    val result: List[Map[String, Any]] = prestoServer
+      .executeQuery("SELECT * FROM presto_spark_db.array_decimal")
+    assert(result.size == 1)
+    for (i <- 0 to 0) {
```

Comment:
result size is only 1. Suggest to remove this redundant loop if not required.

Reply:
done